On musl, time_t is a 64-bit value, even on 32-bit targets.
Therefore, we can't print it with %ld.

diff -upr a/src/log.c b/src/log.c
--- a/src/log.c	2023-04-10 10:05:32.000000000 +0200
+++ b/src/log.c	2023-11-06 13:12:40.168459511 +0100
@@ -3,6 +3,7 @@
 #include <stdio.h>
 #include <sys/resource.h>
 #include <time.h>
+#include <inttypes.h>
 
 #define SECOND 1000000000ul
 
@@ -72,7 +73,7 @@ void log_debug(const char *const fmt, ..
 	va_start(args, fmt);
 	fprintf(
 		stderr,
-		"[%ld.%06ld, %ld.%06ld, %5ld KB][DEBUG]: ",
+		"[%"PRId64".%06ld, %"PRId64".%06ld, %5ld KB][DEBUG]: ",
 		real_time.tv_sec,
 		real_time.tv_nsec / 1000,
 		cpu_time.tv_sec,
